src/topp/CometAdapter.cpp

```diff
@@ -141,43 +141,26 @@ class TOPPCometAdapter :
     setValidStrings_("precursor_error_units", ListUtils::create<String>("amu,ppm,Da"));
     //registerIntOption_("mass_type_parent", "<num>", 1, "0=average masses, 1=monoisotopic masses", false, true);
     //registerIntOption_("mass_type_fragment", "<num>", 1, "0=average masses, 1=monoisotopic masses", false, true);
     //registerIntOption_("precursor_tolerance_type", "<num>", 0, "0=average masses, 1=monoisotopic masses", false, false);
     registerStringOption_(Constants::UserParam::ISOTOPE_ERROR, "<choice>", "off", "This parameter controls whether the peptide_mass_tolerance takes into account possible isotope errors in the precursor mass measurement. Use -8/-4/0/4/8 only for SILAC.", false, false);
     setValidStrings_(Constants::UserParam::ISOTOPE_ERROR, ListUtils::create<String>("off,0/1,0/1/2,0/1/2/3,-8/-4/0/4/8"));
 
-    //Search Enzyme
-    vector<String> all_enzymes;
-    ProteaseDB::getInstance()->getAllCometNames(all_enzymes);
-    registerStringOption_("enzyme", "<cleavage site>", "Trypsin", "The enzyme used for peptide digestion.", false, false);
-    setValidStrings_("enzyme", all_enzymes);
-    registerStringOption_("second_enzyme", "<cleavage site>", "", "The enzyme used for peptide digestion.", false, true);
-    setValidStrings_("second_enzyme", all_enzymes);
-
-    registerStringOption_("num_enzyme_termini", "<choice>", "fully", "Specify the termini where the cleavage rule has to match", false, false);
-    setValidStrings_("num_enzyme_termini", ListUtils::create<String>("semi,fully,C-term unspecific,N-term unspecific"));
-    registerIntOption_("allowed_missed_cleavages", "<num>", 0, "Number of possible cleavage sites missed by the enzyme. It has no effect if enzyme is unspecific cleavage.", false, false);
-    setMinInt_("allowed_missed_cleavages", 0);
-    setMaxInt_("allowed_missed_cleavages", 5);
-
-    registerIntOption_("min_peptide_length", "<num>", 5, "Minimum peptide length to consider.", false);
-    setMinInt_("min_peptide_length", 5);
-    setMaxInt_("min_peptide_length", 63);
-    registerIntOption_("max_peptide_length", "<num>", 63, "Maximum peptide length to consider.", false);
-    setMinInt_("max_peptide_length", 5);
-    setMaxInt_("max_peptide_length", 63);
-
     //Fragment Ions
-    registerDoubleOption_("fragment_bin_tolerance", "<tolerance>", 0.02, "Bin size (in Da) for matching fragment ions. Ion trap: 1.0005, high res: 0.02. CAUTION: Low tolerances have heavy impact on RAM usage. Consider using use_sparse_matrix and/or spectrum_batch_size.", false, true);
-    setMinFloat_("fragment_bin_tolerance", 0.01);
-    registerDoubleOption_("fragment_bin_offset", "<fraction>", 0.0, "Offset of fragment bins scaled by tolerance. Ion trap: 0.4, high res: 0.0.", false, true);
+    registerDoubleOption_("fragment_mass_tolerance", "<tolerance>", 0.3,
```

Comment:
is it a good idea to change to lowers defaults?

Reply:
this is what other adapters have as well (xtandem, for example).
But given the fact that this is a bin-width, one might be off the center, so yeah... lets go with 0.5, which is what is was before.